src/Controller/LtiController.php

```diff
@@ -263,10 +263,19 @@ protected function saveTokenToSession($token)
     protected function saveRequestToSession()
     {
         try {
+            $getParams = $this->request->query->all();
             $postParams = $this->request->request->all();
```

Comment:
@ottenhoff What do you think of shortening this by combining the arrays and then doing just a single loop? I.E. something like
```diff
         try {
             $getParams = $this->request->query->all();
             $postParams = $this->request->request->all();
+            $allParams = array_merge($getParams, $postParams);
 
-            foreach ($getParams as $key => $val) {
-                if (!empty($val)) {
-                    $this->session->set($key, $val);
-                }
-            }
-
-            foreach ($postParams as $key => $val) {
+            foreach ($allParams as $key => $val) {
                 if (!empty($val)) {
                     $this->session->set($key, $val);
                 }
```

Reply:
Yeah, I considered just combining the two arrays, but I decided that would make debugging more difficult in case of an issue where URL params were being sent in addition to POST params. I thought this way was more explicit in demonstrating that POST params would take precedence.

Reply:
Okay, array_merge implemented.